compiler/optimizer/LoopCanonicalizer.hpp

```diff
@@ -256,6 +256,8 @@ class TR_LoopCanonicalizer : public TR_LoopTransformer
    void rewritePostToPreIncrementTestInRegion(TR_RegionStructure *region);
    void rewritePostToPreIncrementTestInBlock(TR::Block *block);
 
+   bool isTrivialArrayOperation(TR::NodeChecklist &visited,TR::Node *node);
```

Comment:
Does this need to exist as a member function? It does not appear to use any instance fields of this class so it could be defined as a static function in the compilation unit.